public/index.html

```diff
@@ -57,25 +58,24 @@
         font-weight: 535;
         font-style: normal;
         font-display: block;
-        font-named-instance: 'Book';
         src: 
-          url(%PUBLIC_URL%/fonts/Basel-Medium.woff) format('woff');
+          url('%PUBLIC_URL%/fonts/Basel-Medium.woff2') format('woff2'),
+          url('%PUBLIC_URL%/fonts/Basel-Medium.woff') format('woff');
       }
 
       @font-face {
         font-family: 'Basel';
         font-weight: 485;
         font-style: normal;
         font-display: block;
-        font-named-instance: 'Book';
         src: 
-          url(%PUBLIC_URL%/fonts/Basel-Book.woff) format('woff');
+          url('%PUBLIC_URL%/fonts/Basel-Book.woff') format('woff2'),
+          url('%PUBLIC_URL%/fonts/Basel-Book.woff') format('woff');
       }
 
       @supports (font-variation-settings: normal) {
         * {
           font-family: 'Basel', sans-serif;
```

Comment:
Why don't we pre-load a variable font for Basel? Is this not available? Are we trying to save bandwidth?

Reply:
Variable font is much higher weight yea thats basically the reason, there was also a weird issue with variable font having a different `$` sign but not as big a reason